buổi-9/bai5/bai-5.js

```diff
@@ -0,0 +1,16 @@
+function rank(){
+    var diemTB=window.prompt('Nhập điểm trung bình: ')
+    if(diemTB>=9){
+        document.write('Hạng A')
+    }
+    if(diemTB>=7 && diemTB<9){
```

Comment:
```suggestion
    if (avg >= 7 && avg < 9) {
```
nhìn kỹ cách đặt dấu cách